### DISCUSSION TITLE
Custom Team Membership

### DISCUSSION THREAD

**Discussion by andreilgeorgescu** [riverwalk-software/astro-test#82] [Q&A] [answered]
@daniel-ciocirlan Instead of having team membership plans of fixed sizes (3, 5, 10, 15), would you prefer to just have something like the following in the pricing section of the the memberships page:

![image](https://github.com/user-attachments/assets/6f23e6ee-7473-4e7b-a633-0c8ba4e58514)

This could be the personal monthly/annual in the white box and the emphasized box could talk about custom memberships for teams. This would clean up the UI and I figure unless the team size happens to be exactly one of those fixed sizes, they will end up contacting you for a custom team membership anyway.

The course pricing sections wouldn't need to change.

**Comment by wkenned1** [comment]
Does Teachable allow you to create a "per seat" membership pricing for any number of teammates? If so, we can simplify the team pack pricing options.

![image](https://github.com/user-attachments/assets/d125a7e9-05a0-4162-adcf-390eed8a2667)

**Comment by daniel-ciocirlan** [comment, marked as answer]
I think we can go with "custom". Even with the clear options on the current pages (3,5,10,15), they still get in touch with me.
